CONTRIBUTING.md

```diff
@@ -119,3 +121,7 @@ Examples:
 
   BREAKING CHANGE: `extends` key in config file is now used for extending other config files
   ```
+
+#### Releases
+
+Please refer to the documentation within JupyterHub's [RELEASE.md](https://github.com/jupyterhub/jupyterhub/blob/main/RELEASE.md).
```

Comment:
Oh wait, this is incorrect - we should refer to https://github.com/jupyterhub/ltiauthenticator/blob/main/RELEASE.md